src/main/java/com/umc/networkingService/domain/member/entity/Member.java

```diff
@@ -75,6 +75,11 @@ public class Member extends BaseEntity {
 
     private String notionLink;
 
+    //포인트 차감
+    public void usePoint(Long usedPoint) {
+        this.remainPoint -= usedPoint;
+    }
+
```

Comment:
기여도 조회가 잦아서, 성능을 위해 contributionPoint 필드를 추가했습니다. 따라서 포인트 사용 시 contributionPoint도 같이 변경해 주면 좋을 거 같아요.
```suggestion
    //포인트 차감
    public void usePoint(Long usedPoint) {
        this.remainPoint -= usedPoint;
        this.contributionPoint += usedPoint;
    }

```